lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,194 @@
 // lms - views - shopping cart
 // ====================
 
+$enrollment-details-width-difference: $baseline*11.85;
```

Comment:
I have to mention, though, that SOL-837 is a story associated only with the shopping cart page. Rest of the pages that are part of the e-commerce feature have their own stories.